requirements_all.txt

```diff
@@ -1625,7 +1625,7 @@ pyuptimerobot==0.0.5
 # pyuserinput==0.1.11
 
 # homeassistant.components.vera
-pyvera==0.3.4
+pyvera==0.3.6
```

Comment:
Bump the version in manifest.json. Then run `script/gen_requirements_all.py` to update this file.